vassal-app/src/main/java/VASSAL/build/module/map/boardPicker/board/RegionGrid.java

```diff
@@ -860,10 +860,10 @@ public void mouseClicked(MouseEvent e) {
               final Action a =
                 new EditPropertiesAction(lastClickedRegion, null, this);
               a.actionPerformed(
-                  new ActionEvent(
-                      e.getSource(),
-                      ActionEvent.ACTION_PERFORMED,
-                      "Edit")); //$NON-NLS-1$
+                new ActionEvent(
+                e.getSource(),
+                ActionEvent.ACTION_PERFORMED,
+                "Edit")); //$NON-NLS-1$
```

Comment:
The indentation went really strange here and in the next few edits to this file.

Reply:
Musta got mangled in somebody's source-prettifier or mergerizer or something. Fixed and checked in.